File: qb-engineer-ui/src/app/features/purchase-orders/models/purchase-order-list-item.model.ts
export interface PurchaseOrderListItem {
  id: number;
  poNumber: string;
  vendorId: number;
  vendorName: string;
  jobId: number | null;
  jobNumber: string | null;
  status: string;
  lineCount: number;
  totalOrdered: number;
  totalReceived: number;
  expectedDeliveryDate: string | null;
  createdAt: string;
}

File: qb-engineer-server/qb-engineer.core/Models/PurchaseOrderListItemModel.cs
namespace QBEngineer.Core.Models;

public record PurchaseOrderListItemModel(
    int Id,
    string PONumber,
    int VendorId,
    string VendorName,
    int? JobId,
    string? JobNumber,
    string Status,
    int LineCount,
    int TotalOrdered,
    int TotalReceived,
    DateTimeOffset? ExpectedDeliveryDate,
    bool IsBlanket,
    DateTimeOffset CreatedAt);
